### DISCUSSION TITLE
Is it possible to downgrade from On-Prem Business to On-Prem Community Editions?

### DISCUSSION THREAD

**Discussion by TheDanHealy** [outline/outline#5991] [General] [answered]
Hi Outline team,

If I'm currently running Outline on-premises with Docker and using a Business license (that's the $4/user license for on-prem one step below enterprise), is it possible to not renew my license and change my container image from `outline-enterprise:latest` to `outline:latest` without any issues?

Thanks

**Comment by tommoor** [comment, marked as answer]
You can switch between editions without issue as long as you're not using the SAML authentication option
